### DISCUSSION TITLE
Short branch length

### DISCUSSION THREAD

**Discussion by Edison2021** [hahnlab/CAFE5#199] [Q&A] [answered]
Hi Ben, I wonder if CAFE 5 can handle branch lengths smaller than 1.0?

Edison

**Comment by benfulton** [comment, marked as answer]
Mathematically, yes; but you should check with an expert in phylogeny (not me!) on the validity of lambdas calculated in this short a time frame.

**Comment by Edison2021** [comment]
Good point! Thanks!

**Comment by YuanQing-KIB** [comment]
Hello, I have the same problem, some nodes in the tree because the branch length is less than 1, causing the expansion and contraction of the gene family are 0. How do you solve this, please

**Comment by benfulton** [comment]
That doesn't seem surprising. A branch length of 1 represents 1 million years, and you wouldn't expect all that much expansion or contraction in a timespan smaller than that.
